ModelBundle/Repository/WorkflowProfileRepository.php

```diff
@@ -46,4 +48,83 @@ public function hasTransition(StatusInterface $fromStatus, StatusInterface $toSt
 
         return $profile instanceof WorkflowProfileInterface;
     }
+
+    /**
+     * @param PaginateFinderConfiguration $configuration
+     *
+     * @return array
+     */
+    public function findForPaginate(PaginateFinderConfiguration $configuration)
```

Comment:
Should we create an abstract class with findForPaginate, countWithFilter, filterSearch and removeMultiple ?